libpimeval/src/pimCmd.h

```diff
@@ -324,11 +306,11 @@ template <typename T> class pimCmdRedSum : public pimCmd
 
 //! @class  pimCmdBroadcast
 //! @brief  Pim CMD: Broadcast a value to all elements
-template <typename T> class pimCmdBroadcast : public pimCmd
+class pimCmdBroadcast : public pimCmd
```

Comment:
Template type is not needed for broadcast, and conversion from specific type to bits is in pimSim.cpp